examples/example_ackRead_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 The Moov Authors
```

Comment:
Does this filename need to be something like `ach_read_example.go`?

Reply:
Good question.  I'm not sure but I used the sort package as an example for the naming convention.

https://golang.org/src/sort/

Reply:
Okay. I'd follow the stdlib as that's kept updated on each release

Reply:
yea, I agree it makes sense.  I was also looking at the go/doc source code and I'm going to make some adjustments based on that as well...